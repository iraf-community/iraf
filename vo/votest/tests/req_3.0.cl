#
#  Query Registry with constraints.

# Set the test description string.
votest.descr = "Query Registry with constraints."

print ("------------------------------------------------------------------")
print ("Req  3.:   Users shall be able to query the VO Registry using the")
print ("           keyword-search services provided by the VO Directory")
print ("	   service.")
print ("------------------------------------------------------------------")


# Execute the test commands.
print ("These requirements are demonstrated specifically by the test scripts")
print ("and more generally by the REGISTRY task itself.")
print ("")
print ("")
