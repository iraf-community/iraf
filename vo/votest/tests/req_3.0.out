------------------------------------------------------------------
Req  3.:   Users shall be able to query the VO Registry using the
           keyword-search services provided by the VO Directory
	   service.
------------------------------------------------------------------
These requirements are demonstrated specifically by the test scripts
and more generally by the REGISTRY task itself.


